src/screens/AddNewItemScreen/index.tsx

```diff
@@ -21,11 +22,14 @@ export const AddNewItemScreen: FC = () => {
   const channelId = useSelector(deviceTokenSelector);
 
   const [text, setText] = useState<string>('');
+  const [date, setDate] = useState(new Date());
+  const [open, setOpen] = useState<boolean>(false);
```

Comment:
you can skip type if you set initial state and this type so simple like boolean, string and so on 

if this type can be null | boolean you should manually set type